estate/models/offer.py

```diff
@@ -0,0 +1,85 @@
+from odoo import models, fields, api
+from odoo.exceptions import UserError
+from datetime import timedelta
+from odoo.tools.float_utils import float_compare
+
+
+class Offer(models.Model):
+    _name = "estate.offers"
```

Comment:
Same as for buildings, make it consistent
```suggestion
    _name = "estate.offer"
```